src/manifest.c

```diff
@@ -567,6 +567,28 @@ static void set_untracked_manifest_files(struct manifest *manifest)
 	}
 }
 
+/* Removes the extracted Manifest.<bundle> and accompanying tar file, as well as
+ * the signature file it exists */
+void remove_manifest_files(char *filename, int version)
+{
+	char *file;
+
+	printf("Warning: Removing corrput Manifest.%s and re-downloading...\n", filename);
+	string_or_die(&file, "%s/%i/Manifest.%s", state_dir, version, filename);
+	unlink(file);
+	free(file);
+	string_or_die(&file, "%s/%i/Manifest.%s.tar", state_dir, version, filename);
+	unlink(file);
+	free(file);
+
+	/* Remove signature if it exists */
+	string_or_die(&file, "%s/%i/Manifest.%s.sig", state_dir, version, filename);
+	if (access(file, F_OK) != -1) {
```

Comment:
Yeah I figured but didn't want to not check for errors. If you want that to stay consistent I can change it.